examples/framesjs-starter/app/debug/page.tsx

```diff
@@ -39,14 +39,13 @@ export default function Page({
     number | null
   >(null);
 
-  const [mockHubContext, setMockHubContext] = useState<FrameActionHubContext>({
-    isValid: false,
+  const [mockHubContext, setMockHubContext] = useState<
+    Partial<FrameActionHubContext>
+  >({
     requesterFollowsCaster: false,
     casterFollowsRequester: false,
     likedCast: false,
     recastedCast: false,
-    requesterVerifiedAddresses: [],
```

Comment:
Not sure what this does. the requester will still have these, and if we're impersonating, then we could provide valid values?

Reply:
can add them as a separate improvements PR along with changing the çastId (fid, hash). none of this is exposed in the UI right now and isn't being handled by the frame action proxy yet